tests/test_api.py

```diff
@@ -248,7 +248,6 @@ def test_get_contexts(self):
         resp = js(self.c.get_contexts("user"))
 
         self.c._request.assert_called_once_with(
-            "GET",
```

Comment:
Yes, you’re right, though odd that it passes when that’s not the method that’s being called In the code 🤔 

good catch

Reply:
I updated the way the mock is called so that instead of
`self.c._request = self.c._request_get_depaginate = MagicMock(return_value=f.read())`
each is called separately.

This is a little less terse, but allows testing that the expected method is called, which I think is worth it.

Reply:
>At some point, might be good to also write a test for the depagination side of things, i.e., test some resource with >1 page of results and test the limit + pagination parameters.
👍